[bug] tauri app with angular  - missing script: "start--"

### Describe the bug

Hello, 

during the creation of my project I had a problem. I created with the command `npm create tauri-app@latest`  an angular application. When I entered afterwards `npm run tauri dev` I got the following output:

```
> tauritest@0.0.0 tauri
> tauri dev

     Running BeforeDevCommand (`npm run start-- --port 1420`)
npm ERR! Missing script: "start--"
npm ERR! 
npm ERR! Did you mean this?
npm ERR!     npm start # Start a package
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm
```




### Reproduction

1.  Run `npm create tauri-app@latest`
2.  Change your project name in the file angular.json. Now `ng serve` should work,
3. run `npm run tauri dev`

### Expected behavior

Running the tauri default app like expected

### Platform and versions

```text
[✔] Environment
    - OS: Windows 10.0.19045 X64
    ✔ WebView2: 119.0.2151.44
    ✔ MSVC: Visual Studio Community 2022
    ✔ rustc: 1.69.0 (84c898d65 2023-04-16)
    ✔ cargo: 1.69.0 (6e9a83356 2023-04-12)
    ✔ rustup: 1.26.0 (5af9b9484 2023-04-05)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 18.12.0
    - npm: 8.19.2

[-] Packages
    - tauri [RUST]: 1.5 (no lockfile)
    - tauri-build [RUST]: no manifest (no lockfile)
    - wry [RUST]: no manifest (no lockfile)
    - tao [RUST]: no manifest (no lockfile)
    - @tauri-apps/api [NPM]: 1.5.1
    - @tauri-apps/cli [NPM]: 1.5.6

[-] App
    - build-type: bundle
    - CSP: unset
    - distDir: ../dist/{% package_name %}
    - devPath: http://localhost:1420/
    - framework: Angular
    - bundler: Webpack
```
```


### Stack trace

_No response_

### Additional context

_No response_

--- a/.changes/angluar.md
+++ b/.changes/angluar.md
@@ -1,6 +1,6 @@
 ---
 "create-tauri-app": "patch"
-"create-tauri-app-node": "patch"
+"create-tauri-app-js": "patch"
 ---
 
 Fix incorrect `beforeDevCommand` in `angluar` template when using `npm` as the package manager.

--- a/.changes/angluar.md
+++ b/.changes/angluar.md
@@ -1,0 +1,6 @@
+---
+"create-tauri-app": "patch"
+"create-tauri-app-node": "patch"
+---
+
+Fix incorrect `beforeDevCommand` in `angluar` template when using `npm` as the package manager.

--- a/packages/cli/fragments/fragment-angular/_cta_manifest_
+++ b/packages/cli/fragments/fragment-angular/_cta_manifest_
@@ -2,13 +2,13 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-License-Identifier: MIT
 
-beforeDevCommand = {% pkg_manager_run_command %} start{% double_dash %} --port 1420
+beforeDevCommand = {% pkg_manager_run_command %} start {% double_dash_with_space %}--port 1420
 beforeBuildCommand = {% pkg_manager_run_command %} build
 devPath = http://localhost:1420
 distDir = ../dist/{% package_name %}
 
 [mobile]
-beforeDevCommand = {% pkg_manager_run_command %} start{% double_dash %} --port 1420 --host $HOST --public-host $HOST
+beforeDevCommand = {% pkg_manager_run_command %} start {% double_dash_with_space %}--port 1420 --host $HOST --public-host $HOST
 
 [files]
 tauri.svg = src/assets/tauri.svg

--- a/packages/cli/src/template.rs
+++ b/packages/cli/src/template.rs
@@ -237,9 +237,9 @@ impl<'a> Template {
         let manifest_template_data: HashMap<&str, &str> = [
             ("pkg_manager_run_command", pkg_manager.run_cmd()),
             (
-                "double_dash",
+                "double_dash_with_space",
                 if pkg_manager == PackageManager::Npm {
-                    "--"
+                    "-- "
                 } else {
                     ""
                 },

